verifier.go

```diff
@@ -420,6 +420,12 @@ func checkMsgName(m string, msgs []MessageElement) bool {
 		if msg.Name == m {
 			return true
 		}
+		// Recursively check for messages within a message.
+		if len(msg.Messages) > 0 {
+			if checkMsgName(m, msg.Messages) {
```

Comment:
The reason I do this `if` is because as soon as we find a true we want to return. But if we find a false, we need to give the rest of the messages a chance to return true. Let me know if this logic is flawed somewhere.